docs/scripts/generate_plugin_docs.py

```
@@ -993,29 +1020,96 @@ def generate_all_docs(self, readme_content: str):
+        sections = [
+            ("Machine Learning Lab", "machine-learning-lab"),
+            ("Visualization Lab", "visualization-lab"),
+        ]
+        for section_name, category in sections:
+            section = self._extract_table_section(readme_content, section_name)
+            if not section:
+                continue
+            for plugin in self._parse_html_table(section, category):
+                slug = plugin.name.split("/")[-1].replace("`", "").strip()
+                if slug in self._SKIP_NAMES:
+                    logger.info(f"Skipping labs feature: {plugin.name}")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)